torch/nn/_functions/thnn/activation.py

```diff
@@ -126,6 +127,56 @@ def backward(self, grad_output):
         return grad_input.mul(-1)
 
 
+class Threshold(Function):
```

Comment:
Can you add a TODO that says it should be removed once we add support for Variable backward in auto-generated THNN functions? While I'll accept this change, because I want this feature to be available quickly, I don't want this to be our long term solution.